src/app/store/reducers/mail.reducers.ts

```diff
@@ -158,13 +159,18 @@ export function reducer(
     }
 
     case MailActionTypes.STAR_MAIL_SUCCESS: {
+      let index: number;
       const listOfIDs = action.payload.ids.split(',');
-      state.mails = state.mails.map(mail => {
+      state.mails = state.mails.map((mail, ind) => {
```

Comment:
what is `ind`?
write either `currentIndex` or just `i`.

Reply:
Okay.